### DISCUSSION TITLE
UI question: how to hide All Items, Colors, Tags buttons v. 1.19.0

### DISCUSSION THREAD

**Discussion by zoda61** [keeweb/keeweb#2220] [General Discussion]
Is there any appearance settings or other method to hide the All Items, Colors, Tags buttons at the top of the tree structure? The aim is to maximize the tree structure area when no need of these buttons.

**Comment by Aetherinox** [comment]
Snap a screenshot and mark off what all parts you're looking to hide. I'll see if I can come up with some type of "Minimalist" mode that makes sense. And decide based on the amount of work when you include it.

v1.19.0 has been nearing the end, the point of the release is to add a few new things but also get us back up to speed with packages, etc. And that's just about done other than a few last minute things.

**Comment by zoda61** [comment]
The marked buttons take a lot of space even when no tags are used, I mean when you have only the 3 main buttons without defined tags. Every new tag steals place. Especially on mobile screens.
![keeweb1](https://github.com/user-attachments/assets/231e4636-1701-4c41-8175-2a374cc2caa4)

**Reply by Aetherinox** [reply]
So you're speaking of just the tags / color options in the top left, and bringing the categories up to meet the top. Just for that, could probably just get away with a toggable button to hide it.

Also, the icons being that close to each label is driving me nuts, and needs fixed to add a gap.

**Reply by zoda61** [reply]
Even the 'All Items' button should be hidable, så the whole category trees touch the top. I do not know what is the best or easiest to do, but the category tree should be always on top, even when these 3 buttons (functions) are visible somewhere or some way.

**Reply by Aetherinox** [reply]
I may opt to just flip them around, and put the cat tree at the top, and tags at the bottom-left as some toggable way.

**Reply by Aetherinox** [reply]
Created an issue, so add anything else there
- https://github.com/keeweb/keeweb/issues/2221

I'll take a look at this once the WebDAV features and password generator tasks are done. Doing the pass gen now.

**Reply by zoda61** [reply]
Thank you!
